auth.go

```diff
@@ -503,6 +544,19 @@ func (x *Central) authenticate(identity, password string) (AuthUser, error) {
 		authTypeCheck = AuthCheckDefault
 	}
 
+	invalidPasswords := x.Stats.UserLoginAttempts[identity]
```

Comment:
In other places in the code, you check first whether UserLoginAttempts is nil, but here you assume it is not. Either always check, or always initialize the map, so that we never have to worry whether it's null or not (I would do the latter)